src/Process/PostDeploy/WarmUp.php

```diff
@@ -111,50 +95,25 @@ public function execute()
      *
      * @return array
      */
-    private function getUrlsForWarmUp(): array
+    public function getUrlsForWarmUp(): array
     {
```

Comment:
This must be private

Reply:
Why? Private methods cannot be unit tested.

Reply:
You should not create API methods (which are protected and public) just for unit testing. Test original `execute` method or decouple class if it's to complex